sketch.js

```diff
@@ -1,223 +1,69 @@
-//Kanye: Me vs We 
+//Kanye: Me vs We
 
-function setup() {
-  createCanvas(1120,600);
-  background(51);
-}
+var meWords = ["I","me","Kanye","West"]
+var weWords = ["you","we","youre","they", "he","she","them"]
 
-function draw() {
-  fill('hotpink');
-  textSize(25);
-  textFont("Helvetica")
-  text("Kanye: Me vs We", 40, 50);
-  textSize(12);
-  //text("an infographic", 50, 80);
-  
-  kanye();
-  albumNames();
-  splitWords();
+// Splits a string into an array of words
+var splitWords = function(aStringOfLyrics) {
+  return aStringOfLyrics.toLowerCase().split(" ");
 }
 
+// Takes in an array of words and gives frequency
```

Comment:
Instead of using an `array` I'm using an object. Rather than just having an index that's a number you can specify an index as a `key`. Here I'm just setting a key for every word. Later I increment each value.